### PR TITLE
Add ef core database drop support.

### PR DESCRIPTION
**Pull request by EngincanV**
Resolves https://github.com/enisn/AbpDevTools/issues/1

Screenshot 2023-12-02 171546

If you think these outputs are too much, I can convert them to the live table view, as you did for other commands. Just let me know @enisn

### CODE REVIEW

**Comment by enisn** [comment]
Thanks for your contribution 🎉 🙏
I'll check and try to merge asap.

> If you think these outputs are too much, I can convert them to the live table view, as you did for other commands. Just let me know

Yeah, an interactive table might be useful. Or a report at the end of the process might be enough

File: src/AbpDevTools/Commands/DatabaseDropCommand.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    {
+        if (string.IsNullOrEmpty(WorkingDirectory))
+        {
+            WorkingDirectory = Directory.GetCurrentDirectory();
+        }
+
+        var efCoreProjects = Directory.EnumerateFiles(WorkingDirectory, "*.csproj", SearchOption.AllDirectories)
+            .Where(x => x.EndsWith("EntityFrameworkCore.csproj"))
+            .Select(x => new FileInfo(x))
+            .ToList();
```
Comment (enisn) [inline src/AbpDevTools/Commands/DatabaseDropCommand.cs:33]: We can show a spinner while searching files, when the command is executed in a very root folder like `D:\`, the process can take some times and user shouldn't think it's stuck

Here is an example:
https://github.com/enisn/AbpDevTools/blob/f22b39401acca0899108aff2c7346b9a598f69f3/src/AbpDevTools/Commands/BuildCommand.cs#L122-L140

File: src/AbpDevTools/Commands/DatabaseDropCommand.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            
+        var cancellationToken = console.RegisterCancellationHandler();
```
Comment (enisn) [inline src/AbpDevTools/Commands/DatabaseDropCommand.cs:35]: When a cancellation requested by `CTRL` + `C`, the process can stay hanging. We should kill running process with it's children processes together. Here is an example:
https://github.com/enisn/AbpDevTools/blob/f22b39401acca0899108aff2c7346b9a598f69f3/src/AbpDevTools/Commands/BuildCommand.cs#L115

Comment (enisn) [reply src/AbpDevTools/Commands/DatabaseDropCommand.cs:35]: Ok, they're running one by one, and throıwn exception by cancellationtoken prevents handling processes ✅ 👍

File: src/AbpDevTools/Commands/DatabaseDropCommand.cs (hunk 1 above)
Comment (enisn) [reply src/AbpDevTools/Commands/DatabaseDropCommand.cs:33]: abpdevtools-spinner

Comment (EngincanV) [reply src/AbpDevTools/Commands/DatabaseDropCommand.cs:33]: Sure, I'll add spinner 👍

**Review by enisn — APPROVED** [review]
Thanks for your contribuion @EngincanV  🙏

**Pull request merged**